docs/section5/Rule5-8.md

```diff
@@ -0,0 +1,45 @@
+
+# Envelope - Rule 5-8  
+
+**Rule ID:** 5-8  
+**Rule Description:**  Fenestration U-factors for new buildings, existing buildings, and additions shall match the appropriate requirements in Tables G3.4-1 through G3.4-8 for the applicable glazing percentage for Uall.  
+**Rule Assertion:** Baseline RMR = expected value  
```

Comment:
Typo with 'Uall', should be 'Wall'

Reply:
Should say 'Fenestration U-factors for baseline' and not 'Fenestration U-factors for new buildings, existing buildings, and additions'

Reply:
Uall is the term used in the table (the all is a subscript), it is not a typo.